remodel/connection.py

```diff
@@ -1,5 +1,5 @@
 
-import rethinkdb as r
+import rethinkdb
```

Comment:
Basically because we need `rethinkdb.__version__` and because the module isn't used more than once I considered it acceptable so I didn't have  to do something like 
`from rethinkdb import __version__ as rethinkdb_version`. But I'm definitely not very opinionated here. 

Basically came down to personal biases in this case. If you prefer that I do two imports instead I will definitely do that.